### DISCUSSION TITLE
Allow link discussion item in Project board beta

### DISCUSSION THREAD

**Discussion by JayZeeDesign** [community/community#11223] [Projects and Issues]
Discussion is perfect for gather ideas & future improvements, but it can't be linked to project board, which means we need to double handling the work

**Comment by armandgrillet** [comment]
Adding to this, some repositories like [Grafana](https://github.com/grafana/grafana) use discussions as the place for feature requests. Being able to link discussions in project boards would be useful in such cases.

**Comment by choldgraf** [comment]
Just providing my support for this as well. Many teams may wish to track an ongoing discussion as part of a Project before they decide if / how to convert it into an actionable issue. It would be helpful if they could both be tracked in the same space.

**Comment by SMR39** [comment]
This would be amazing to have

**Comment by reeserj** [comment]
Chiming in that we'd love this feature too

**Comment by svengreb** [comment]
This is something I‘d highly appreciate!
Discussions have been introduced, among the overall community aspect, as a feature to also help maintainers to reduce the maintenance overhead of to having to distinguish between actual “issues“ and support/help requests in reported issues per repository. The ID of a discussion even shares the same sequential number with issues (last segment in a discussion URL, e.g. this one is `11223`). For me this is a clear indicator that both should be treated the same. Other metadata are also labels and everything else from the repository linked to the organization-wide discussions.

In one of my projects I encourage users to [post help & support requests][1] as well as [suggestions & ideas][2] in the organization-wide discussions (yes, the organization is currently in a migration phase so no posts yet), but not allowing me to manage them in a project board, like before with repository issues, is another overhead in the daily notifications.

I guess there are many larger projects, e.g. [Next.js‘s discussions][3], that welcome this feature to get one step closer to an universal way of managing projects.

[1]: https://github.com/orgs/nordtheme/discussions/categories/help-support
[2]: https://github.com/orgs/nordtheme/discussions/categories/suggestions-ideas
[3]: https://github.com/orgs/vercel/discussions

**Comment by michael-hawker** [comment]
We also use Discussions to track proposals and feature requests, so being able to add them to a board for tracking and triage or proposal stage process separating would be great.

When issues/PRs are so tightly coupled, it's a little odd that discussions also don't just fit in that primary-entity type bucket for all these features.

Having temporary placeholder tasks in a board is great, but if there's a discussion already, I'd much rather link to that.

**Comment by AnthonyMichaelTDM** [comment]
I also agree that this would be a great feature

**Comment by rlancejohnson** [comment]
We are starting to use discussions for feature requests as well. This works great to make decisions and start to create issues from that discussion, but I have noticed that trying to keep track of discussions like this with the current functionality available is really difficult. I am starting to realize that in order to keep this going, linking these discussions to projects so that we can manage order of importance, backlog, etc. is going to be essential. We may have to go back to draft issues until this is implemented, but discussions are really nice for many reasons to handle feature requests.

**Comment by Jungal10** [comment]
Just setting up for my working group and I think this would be a great feature

**Comment by nicksergeant** [comment]
Would love this also. Discussions are useful as larger work items because you can have threaded conversations in one place. We use them for larger projects even though they're not really intended to be used as work items.

So it would be useful to add a Discussion to a project board in this context, as well as being able to reference feedback from the community in your Project without having to create an issue that says essentially the same thing as the discussion.

**Comment by opr** [comment]
This feature would be amazing to have! I would find it very useful to link discussions in a project board.

We need to track and assign certain discussions to certain team members, so assigning them on a project board would be helpful for that.

**Comment by agneszitte** [comment]
+1 it will be so helpful to have this, please
Like many others being able to add discussions to a Project for tracking and triage is a must for us
cc @azenMatt, @evi-liu

**Comment by sumau** [comment]
Adding discussions to projects would also allow us to assign different status to discussions as highlighted in #3775

**Comment by SoumayaMauthoorMOJ** [comment]
Related https://github.com/orgs/community/discussions/128944

**Comment by chodid** [comment]
This is a major annoyance when not just using the code-functionalities but also relying on GitHub for project management.
Issues, due to their linear structure, are not well suited for discussions. The better suited discussions on the other hand don't show up in the project board and could get forgotten if not regularly re-checked and revived.
Is there any way to get the GitHub team to at least take proper notice of this issue and ask for a statement?

**Reply by EamonNerbonne** [reply]
Yeah, in fact, even for many unclear (plain-english) issues github issues are less well-suited than discussions. Threading and voting is often useful, and the lack of those features in github issues means anything without a clear-cut resolution path often devolves into a mess.

**Comment by im-chabani** [comment]
adding my support to this, it would be great !

**Comment by Lee-W** [comment]
+1 for this
